app/forms/placement_form.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class PlacementForm
```

Comment:
I think we need to use https://github.com/DFE-Digital/register-trainee-teachers/blob/b076b8ce197ba29b5e8ca4679e59dd15d68844cf/app/forms/trainee_form.rb#L3 which is the base for all section-y forms.